### ISSUE TITLE
Start the server and open browser from the desktop shortcut 

### ISSUE THREAD

**Issue by radinamatic** [learningequality/kolibri-installer-windows#23]
_Amended with input from @indirectlylit_

In order to make the starting the server as seamless as possible, we agreed to start it just by double-clicking the desktop shortcut.

If I'm not mistaken, this does not necessarily mean eliminating the `Kolibri.exe` launcher, but just alters a bit its workflow:

## Starting Kolibri

1. User double-clicks the desktop shortcut.
2. Kolibri icon appears in the task tray.
3. Server starts and notification bubble `Kolibri is starting, please wait...` appears.
4. If server doesn’t start within 5 seconds, a notification bubble `Kolibri is still starting, please wait...` appears. Repeat the notification every 5 seconds.
5. As soon as the server is running, the default browser opens with the server IP.

## Stopping Kolibri

`Exit Kolibri` would stop the server and exit the task tray - this will leave the browser open, but I don't think that we should close it because user might have other tabs opened.

##  Task tray options

1. `Start server` and `Stop server` options are superfluous  with this new workflow, and they should be eliminated.
2. We are keeping the `Load in browser` option.
3. For now we are keeping the auto-start on boot option with this wording: `Run Kolibri on system start`
4. Shorten the `Exit Kolibri` to simple `Exit`, which stops the server and closes the task tray icon.

### To sum it up - new options

> Load in browser
> Run Kolibri on system start
> Exit

## Error handling

This should cover user feedback when things don’t work, e.g. when Kolibri server dies or is killed from the command line. One option would be to:

1. Show the notification bubble `Kolibri has stopped... You can restart it from the desktop shortcut.` when the server gets shut down by an external method (either crash or manual shutdown).
2. Task tray icon disappears.

**Comment by indirectlylit** [comment]
I believe this is all addressed!
